sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -1,20 +1,8 @@
+import { SaveStudentApplicationDto } from "@/types";
 import HttpBaseClient from "./common/HttpBaseClient";
 
 export class ApplicationApi extends HttpBaseClient {
-  public async createApplication(data: any): Promise<any> {
-    try {
-      return await this.apiClient.post(
-        "application",
-        data,
-        this.addAuthHeader(),
-      );
-    } catch (error) {
-      this.handleRequestError(error);
-      throw error;
-    }
-  }
-
-  public async getApplicationData(applicationId: any): Promise<any> {
+  public async getApplicationData(applicationId: number): Promise<any> {
```

Comment:
fyi-I have edited this line with an interface in my current PR

Reply:
Sounds good, this is not part of this PR.